Gemfile.tt

```diff
@@ -22,6 +22,7 @@ gem "rack-canonical-host", "~> 0.2.3"
 group :development do
   gem "bundler-audit", require: false
   gem "brakeman", require: false
+  gem "faker", require: false
```

Comment:
Probably this should be in the `development, test` group since it's used in both development and for factory generation in tests. _Most_ of the time we'd get away with this since a plain `bundle install` will install all groups anyway, but since it's used for dev and test we should probably have it in the correct group.